arcgis-ios-sdk-samples/Content Display Logic/ContentPList.plist

```diff
@@ -818,103 +818,103 @@
 				<key>storyboardName</key>
 				<string>GORenderers</string>
 				<key>descriptionText</key>
-				<string>You can add graphics to a map using the new Graphics Overlay</string>
+				<string>Add graphics with renderer to a map to change the style of all graphics in a graphics overlay.</string>
```

Comment:
Thanks for pointing that out. I deliberately made small adjustments to the description text so that it matches the others which start with a verb. I decide to change this line to 
```
Add graphics with renderer to a map to change the style of all graphics in a graphics overlay by referencing a single symbol style.
```